packages/awslint/lib/rules/construct.ts

```diff
@@ -99,16 +93,9 @@ constructLinter.add({
     }
 
     const expectedParams = new Array<MethodSignatureParameterExpectation>();
-
-    let baseType;
-    if (process.env.AWSLINT_BASE_CONSTRUCT && !CoreTypes.isCfnResource(e.ctx.classType)) {
-      baseType = e.ctx.core.baseConstructClass;
-    } else {
-      baseType = e.ctx.core.constructClass;
-    }
```

Comment:
This check was redundant because `e.ctx.core.baseConstructClass` and `e.ctx.core.constructClass` are the same.